src/app/routes/getInitialData/mediapage/index.test.js

```diff
@@ -0,0 +1,7 @@
+import getMediaPageInitialData from '.';
+
+describe('getMediaPageInitialData', () => {
+  it('returns a promise', async () => {
+    expect(getMediaPageInitialData({ service: 'amharic' }).then).toBeDefined();
```

Comment:
Whats the `.then`? What are you expecting to be defined?

Reply:
`getInitialData` just needs to return a promise for the router for now

This is another implementation stub - the fixtures that `getInitialData` will `get` is separated out into another issue

Reply:
I'm pretty sure this will pass with or without the `.then` as all we are checking is that something is defined. 

I think something like the following would suffice for testing the stub: 
```
const mockData = {
  status: 200,
  service: 'amharic',
};

describe('getMediaPageInitialData', () => {
  it('returns a promise', async () => {
    expect(getMediaPageInitialData({ service: 'amharic' }).resolve()).toBe(
      mockData,
    );
  });
});

```

Reply:
Actually tired this locally and it looks like the function returns an empty object `{}` 🤔

Reply:
As we are just testing the stub I think awaiting the response of the promise is fine until we have fixture/real data: 

```

const mockData = { service: 'amharic', status: 200 };

describe('getMediaPageInitialData', () => {
  it('returns a promise', async () => {
    expect(await getMediaPageInitialData({ service: 'amharic' })).toEqual(
      mockData,
    );
  });
});
```

Reply:
Shouldn't be returning an empty object, and the unit tests were passing 😕 

![Screen Shot 2019-07-24 at 11 08 17](https://user-images.githubusercontent.com/8601460/61785340-6b4f3a00-ae03-11e9-844f-62f92670e061.png)

Updated to the async test as suggested :)